contrib/storage-http/README.md

```diff
@@ -29,41 +31,193 @@ The required options are:
 * `timeout`:  Sets the response timeout in seconds. Defaults to `0` which is no timeout.
 * `connections`:  This field contains the details for individual connections. See the section *Configuring API Connections for Details*.
 
-You can configure Drill to work behind a corporate proxy. Details are listed below. 
+You can configure Drill to work behind a corporate proxy. Details are listed below.
 
 ### Configuring the API Connections
 
 The HTTP Storage plugin allows you to configure multiple APIS which you can query directly from this plugin. To do so, first add a `connections` parameter to the configuration
 . Next give the connection a name, which will be used in queries.  For instance `stockAPI` or `jira`.
 
-The `connection` can accept the following options:
-* `url`: The base URL which Drill will query. You should include the ending slash if there are additional arguments which you are passing.
-* `method`: The request method. Must be `get` or `post`. Other methods are not allowed and will default to `GET`.
-* `headers`: Often APIs will require custom headers as part of the authentication. This field allows you to define key/value pairs which are submitted with the http request
-.  The format is:
+The `connection` can accept the following options.
+
+#### URL
+
+`url`: The base URL which Drill will query.
+
+`requireTail`: Set to `true` if the query must contain an additional part of the service
+URL as a table name, `false` if the URL needs no additional suffix other than that
+provided by `WHERE` clause filters. (See below.)
+
+If your service requires parameters, you have three choices. Suppose your connection is called
+`myConn`. First, can include them directly in your URL if the parameters a fixed for a given
+service:
+
+```json
```

Comment:
I think this notation does really work with markdown, please see doc display:
https://github.com/paul-rogers/drill/blob/a4226a06361bc136427797f542c401e05d608c5f/contrib/storage-http/README.md

Reply:
Actually, it does work. The link I included was to an earlier version that had formatting errors. (I just learned that the link does not update when I push an update. Changed it by hand.) [New version](https://github.com/apache/drill/blob/0c01fb619d869c9cc0fd8ee094bc4ddd9042b552/contrib/storage-http/README.md).

To be honest, I didn't know about this feature until I saw what Charles had done. But, [it does work](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code) and [many renderers](https://highlightjs.org/static/demo/) are supported.

Reply:
Great, new version looks much better.